### PR TITLE
Hamming

### PR DESCRIPTION
**Pull request by qjd2413**

### CODE REVIEW

**Comment by qjd2413** [comment]
I need to clean up the compare rule, I think I can do it better

File: exercises/hamming/hamming.example [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    string_length(Str1, Length),
+    string_length(Str2, Length),
+    string_chars(Str1, List1),
+    string_chars(Str2, List2),
+    hamming_distance_cmp(List1, List2, Distance).
+
+hamming_distance_cmp([], [], 0) :- !.
+hamming_distance_cmp([Char1|Chars1], [Char2|Chars2], Distance) :-
+    hamming_distance_cmp(Chars1, Chars2, NextDistance),
+    (Char1 == Char2 ->
```
Comment (qjd2413) [inline exercises/hamming/hamming.example]: @parkerl what do you think of this style? the style guide offered a few other options.

**Comment by parkerl** [comment]
I'm good with that style. Are you ready for this to be merged?

**Comment by parkerl** [comment]
Manually merged. Thanks! 🎉

**Pull request closed without merging**